src/functionalTest/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/util/AuthorizationHeadersProvider.java

```diff
@@ -51,4 +51,27 @@ public Headers getLegalRepresentativeAuthorization() {
         );
     }
 
+    public Headers getCaseOfficerAuthorization() {
+
+        MultiValueMap<String, String> tokenRequestForm = new LinkedMultiValueMap<>();
```

Comment:
This has been handled on RIA-3569. Refactored the code. Good idea to take those changes at a later stage

Reply:
It will be resolved when we merge it. Since it is still not ready for deployment I did not want to create a dependency (just in case)|.